### PR TITLE
Added categorized list support for showValidationErrors

### PR DESCRIPTION
**Pull request by sz1521**

### CODE REVIEW

File: src/lib/components/00-atoms/TextBox/index.js [hunk 1]
```diff
@@ -119,6 +119,8 @@ const TextBox = props => {
     }
   }, [props.value]); // If value is added the component won't work.
 
+  console.log("props", props);
```
Comment (rapitkan) [inline src/lib/components/00-atoms/TextBox/index.js]: Remove this, please.

File: src/lib/components/02-organisms/CategorizedListRoot/CategorizedList/index.js [hunk 2]
```diff
@@ -760,6 +752,8 @@ const CategorizedList = React.memo(
                   showCategoryTitles={props.showCategoryTitles}
                   onChangesUpdate={props.onChangesUpdate}
                   removeChangeObject={props.removeChangeObject}
+                  showValidationErrors={showValidationErrors}
+                  requiredMessage={props.requiredMessage}
```
Comment (rapitkan) [inline src/lib/components/02-organisms/CategorizedListRoot/CategorizedList/index.js]: AFAIK requiredMessage isn't CategorizedList level property. Every component that is used by the CategorizedList have their own requiredMessage property. That's why requiredMessage={props.requiredMessage} should be removed.

File: src/lib/components/02-organisms/Difference/index.js [hunk 3]
```diff
@@ -43,15 +43,14 @@ const Difference = ({
   const readonly =
     R.path(["component", "properties", "isReadOnly"], payload) || isReadOnly;
 
-  const isValid = isValueValid(required, value);
+  // const isValid = isValueValid(required, value);
```
Comment (rapitkan) [inline src/lib/components/02-organisms/Difference/index.js]: If isValid is not needed let's remove the unused code.

File: src/lib/components/02-organisms/CategorizedListRoot/CategorizedList/index.js (hunk 2 above)
Comment (sz1521) [reply src/lib/components/02-organisms/CategorizedListRoot/CategorizedList/index.js]: Yes it does need it.

**Pull request merged**